### PR TITLE
[#465] Separate documentation of RESTful resources for device registry.

### PR DESCRIPTION
**Pull request by sysexcontrol**
Currently, the setup description of the device registry in the User Guide
also contains the description of the RESTful resources that are offered
for convenience. This overloads the description and leads to a very
long document for two different things. It is separated into two pages
now.

Signed-off-by: Karsten Frank <Karsten.Frank@bosch-si.com>

### CODE REVIEW

**Comment by sysexcontrol** [comment]
This PR serves as a preparation to include the Tenant API resources as well.

File: site/content/user-guide/device-registry-http.md [hunk 1]
```diff
@@ -0,0 +1,348 @@
++++
+title = "Device Registry - RESTful resources"
+weight = 450
```
Comment (sophokles73) [inline site/content/user-guide/device-registry-http.md]: have you checked if this is in the correct range?

Comment (sysexcontrol) [reply site/content/user-guide/device-registry-http.md]: Thanks for the comment - yes, it renders it to the very end and leaves a larger gap for possible future page inserts (the setup pages list first from 245 - 390, with unequal distances between the single pages). So I thought 450 might be a reasonably flexible weight.
WDYT?

Comment (sophokles73) [reply site/content/user-guide/device-registry-http.md]: That won't work because the *Developer Guide* pages start at 395 according to `config.toml`

Comment (sysexcontrol) [reply site/content/user-guide/device-registry-http.md]: Ah thanks, I missed that (as you see ;-))!
So then I will condense the pages in the user guide regarding their weight, so we are clean again.
Will provide an update soon..

**Comment by sysexcontrol** [comment]
@sophokles73 : Page weights now condensed for the user guide.

**Pull request merged**